src/main/java/frc/robot/subsystems/Swerve/SwerveSubsystem.java

```diff
@@ -153,22 +151,19 @@ public Command followPathCommandTeleop(String pathName, double xVelocity, double
 				() -> this.getChassisSpeedsTeleop(xVelocity,yVelocity,rotationalVelocity),// ChassisSpeeds supplier. MUST BE ROBOT RELATIVE
                 this::drive, // Method that will drive the robot given ROBOT RELATIVE ChassisSpeeds
                 new HolonomicPathFollowerConfig( // HolonomicPathFollowerConfig, this should likely live in your Constants class
-                        new PIDConstants(1.5, 0.0, 0.0), // Translation PID constants
-                        new PIDConstants(2.5, 0.0, 0.0), // Rotation PID constants
+                        new PIDConstants(driveP, driveI, driveD), // Translation PID constants
+                        new PIDConstants(turnP, turnI, turnD), // Rotation PID constants
                         SwerveConstants.maxSpeed, // Max module speed, in m/s
-                        Units.feetToMeters(1), // Drive base radius in meters. Distance from robot center to furthest module.
-                        new ReplanningConfig() // Default path replanning config. See the API for the options here
+                        SwerveConstants.driveBaseRadius, // Drive base radius in meters. Distance from robot center to the furthest module.
+                        new ReplanningConfig() // Default path re-planning config. See the API for the options here
                 ),
                 () -> {
                     // Boolean supplier that controls when the path will be mirrored for the red alliance
                     // This will flip the path being followed to the red side of the field.
                     // THE ORIGIN WILL REMAIN ON THE BLUE SIDE
 
                     var alliance = DriverStation.getAlliance();
-                    if (alliance.isPresent()) {
-                        return alliance.get() == DriverStation.Alliance.Red;
-                    }
-                    return false;
+                    return alliance.filter(value -> value == DriverStation.Alliance.Red).isPresent();
```

Comment:
The original code is more readable.
This is twice in the same file it can be a helper method